src/services/EcoBridge/Socket/Socket.utils.ts

```diff
@@ -149,3 +249,16 @@ export const VERSION = {
   minor: 0,
   patch: 0,
 }
+
+export const SOCKET_LISTS_URL =
+  'https://raw.githubusercontent.com/SwaprDAO/swapr-ecobridge-socket-lists/master/lists/socketList-bidirectional.json'
+
+// Pairs all provided chains
+export const socketSupportedChains = (supportedChains: ChainId[]) =>
+  supportedChains
+    .map((v, i) => supportedChains.slice(i + 1).map(w => [v, w]))
+    .flat()
+    .map<SupportedChainsConfig>(([from, to]) => ({
+      from,
+      to,
+    }))
```

Comment:
do we have flatmap? Does it make sense to use it here?
(Optional change)

Reply:
Honestly - copied somewhere from github. It pairs chains so we won't need to do it manually and it works. If you have better solution then feel free to commit a change :)

Reply:
```suggestion
    .flatMap((v, i) => supportedChains.slice(i + 1).map(w => [v, w]))
    .map<SupportedChainsConfig>(([from, to]) => ({
      from,
      to,
    }))
```

Reply:
Looks good, applied